src/kubectlUtils.ts

```diff
@@ -339,19 +341,20 @@ export async function getResourceAsJson<T extends KubernetesResource | Kubernete
  * Parse column based output which is seperated by whitespace(s) from kubectl or similar sources
  * for example, kubectl get po
  * @param lineOutput raw output with headers from kubectl or similar sources
+ * @param parsingRegex regex to parse the columns
  * @return array of objects with key as column header and value
  */
-export function parseLineOutput(lineOutput: string[]): { [key: string]: string }[] {
+export function parseLineOutput(lineOutput: string[], parsingRegex: RegExp): { [key: string]: string }[] {

```

Comment:
As this is now used for both kubectl and Helm, should we move it to a, say, `outputUtils` or `parsing` file?